content/6_Advanced/Treaps.mdx

```diff
@@ -147,50 +178,64 @@ ll ans, inv;
 
 typedef struct tnode* pt;
 struct tnode {
-	int pri, val; pt c[2]; // essential
-	int sz; // for range queries
-	tnode (int _val) {
-		pri = rng(); val = _val;
-		sz = 1; c[0] = c[1] = NULL;
+	int pri, val;
+	pt c[2];  // essential
+	int sz;  // for range queries
+	tnode (int val) {
+		pri = rng();
+		val = val;
+		sz = 1;
+		c[0] = c[1] = NULL;
 	}
 };
-int getsz(pt x) { return x?x->sz:0; }
+
+int getsz(pt x) { return x ? x->sz : 0; }
```

Comment:
expand this

Reply:
What happened to braces or one line?

Reply:
maybe because functions can't not have braces?

Reply:
just change the code, I didn't even make this